lib/modules/showImages.js

```diff
@@ -1400,13 +1400,16 @@ function setMediaMaxSize(media) {
 }
 
 function addDragListener({ media, atShiftKey, onStart, onMove }) {
-	let isActive, hasMoved, lastX, lastY;
+	let isActive, isReleased, hasMoved, lastX, lastY;
 
 	const handleMove = frameDebounce(e => {
-		if (!e.movementX && !e.movementY) {
+		if (
+			process.env.BUILD_TARGET !== 'safari' && // Safari lacks e.movement(X|Y)
```

Comment:
Yeah this prevents a click from opening the image in Safari. May have to add a `mousedown` listener to set the initial `lastX` and `lastY` and use `e.clientX - lastX` instead of `e.movementX`, if that would be equivalent.